src/anyon/anyon.jl

```diff
@@ -127,10 +131,11 @@ Dict{String, Int64} with 1 entry:
 ```
 """
 function transpile_and_run_job(
-    qpu::AnyonQPU, 
+    qpu::AnyonQPU,
     circuit::QuantumCircuit,
     num_repetitions::Integer;
-    transpiler::Transpiler=get_transpiler(qpu)
+    transpiler::Transpiler=get_transpiler(qpu),
+    status_request_throttle::Function=status_request_throttle
```

Comment:
Sorry, still learning Julia :sweat_smile: 

I thought adding it just to the `run_job` and `transpile_and_run_job` overloads that take `AnyonQPU` was how to do what you're asking... 

I guess the way to do it would be to add a field to the `AnyonQPU` struct, initialize it in the constructors, and call it from `run_job`... is that what you had in mind?

Reply:
As discussed on our call, `run_job` would ideally be a part of the `AbstractQPU` API so divergence between `run_job(::AnyonQPU, ...)` and `run_job(::VirtualQPU, ...)` is not what we want.

I'll add the throttle as a field in the `AnyonQPU`.

I've filed #229 track the task of adding `run_job` to the `AbstractQPU` interface.